app/src/main/java/org/mozilla/fenix/perf/StartupFrameworkStartMeasurement.kt

```diff
@@ -68,7 +68,13 @@ internal class StartupFrameworkStartMeasurement(
         //
         // Similarly, we round app init to a whole integer tick value because process start only
         // comes in integer ticks values.
-        val processStartTicks = stat.getProcessStartTimeTicks(Process.myPid())
+        val processStartTicks =
+            try {
+                stat.getProcessStartTimeTicks(Process.myPid())
+            } catch (e: NegativeArraySizeException) {
```

Comment:
Why are we catching `NegativeArraySizeException`? iiuc, the error is `FileNotFoundException`: https://sentry.prod.mozaws.net/operations/fenix/issues/8293980/

Reply:
Oh, sorry, I think I merged the wrong branch, I was testing throwing random exceptions on another.